### ISSUE TITLE
If caniuse-lite isn't in package.json directly, `yarn info` fails

### ISSUE THREAD

**Issue by osdiab** [browserslist/browserslist#616]
If `caniuse-lite` is included as as transitive dependency of some other package like NextJS, I'm finding that when I run `npx browserslist@latest --update-db` then it fails with this error:

```
Command failed: yarn info caniuse-lite --json
```

I use Yarn 3.0.1. When I add `caniuse-lite` directly, then instead of that error, I get:

```
Cannot read property 'version' of undefined
```

For reference, `yarn info caniuse-lite --json` returns:

```
{"value":"caniuse-lite@npm:1.0.30001252","children":{"Version":"1.0.30001252"}}
```

And if this were to work without `caniuse-lite` being installed as a direct dependency, the `yarn info` command just needs the `--recursive` flag:

```
yarn info caniuse-lite --recursive --json
```

Yielding the same result as above.

It seems that the `update-db` command doesn't work properly for Yarn Berry. I presume that even if I got past this, it would fail later due to #577 because it doesn't use `yarn up -R caniuse-lite` as suggested there.

**Comment by ai** [comment]
The fix was released in 4.17.

**Comment by KVROHITH098** [comment]
Could u use the same but a little different type of command. Because I am working on this too.
